bellows/ash.py

```diff
@@ -393,6 +394,22 @@ def close(self):
             self._transport.close()
             self._transport = None
 
+    def _reject_frame(self) -> None:
+        """Send NAK and enter reject condition if not already rejected."""
+        if self._ncp_state == NcpState.CONNECTED and not self._in_reject_condition:
```

Comment:
nit I've learned by this being corrected in every second Core PR, as it's slightly faster:
```suggestion
        if self._ncp_state is NcpState.CONNECTED and not self._in_reject_condition:
```
(just leave it like this)